.github/workflows/ci.yml

```diff
@@ -0,0 +1,20 @@
+name: ci
+on: [pull_request]
+jobs:
+  build-and-test:
+    runs-on: ${{ matrix.os }}
+    env:
+      USE_BAZEL_VERSION: ${{ matrix.bazel_version }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, macos-10.15]
```

Comment:
Why do we want to run this on both ubuntu and macos? IIRC macos is a lot more expensive than ubuntu. I'm also curious why we want to run it with 2 different bazel versions

Reply:
Because we want the rules to be usable in linux and MacOS and we officially support two bazel versions. MacOS is generally pricier, but since this is a public project, it's free as long as we don't abuse it.